### PR TITLE
Sa/cleanup2

### PR DESCRIPTION
**Pull request by genestack-solomatin**

### CODE REVIEW

File: genestack_client/settings/genestack_config.py [hunk 1]
```diff
@@ -73,7 +73,7 @@ def add_user(self, user, save=True):
 
     def set_default_user(self, user, save=True):
         if user.alias not in self.__users:
-            raise GenestackException('User %s is not present in config users' % user.alias)
+            raise GenestackException('User "%s" is not present in config users' % user.alias)
```
Comment (klas-genestack) [inline genestack_client/settings/genestack_config.py]: > in config users

What?  Maybe in "config file"?

File: genestack_client/settings/genestack_config.py [hunk 2]
```diff
@@ -58,13 +58,13 @@ def remove_user(self, user):
...
-            print "Error while deleting user password for %s: %s" % (user.alias, e)
+            sys.stderr.write('Error while deleting user password for %s: %s\n' % (user.alias, e))
 
     def add_user(self, user, save=True):
         if not user.alias:
-            raise GenestackException("Cant add user without alias to config")
+            raise GenestackException('Cant add user without alias to config')
         if user.alias in self.__users:
-            raise GenestackException("User alias %s is already present" % user.alias)
+            raise GenestackException('User alias %s is already present' % user.alias)
```
Comment (klas-genestack) [inline genestack_client/settings/genestack_config.py]: 'User alias "%s" already exists'

File: genestack_client/settings/genestack_config.py [hunk 3]
```diff
@@ -58,13 +58,13 @@ def remove_user(self, user):
             if keyring.get_password(_PASSWORD_KEYRING, user.alias):
                 keyring.delete_password(_PASSWORD_KEYRING, user.alias)
         except Exception as e:
-            print "Error while deleting user password for %s: %s" % (user.alias, e)
+            sys.stderr.write('Error while deleting user password for %s: %s\n' % (user.alias, e))
 
     def add_user(self, user, save=True):
         if not user.alias:
-            raise GenestackException("Cant add user without alias to config")
+            raise GenestackException('Cant add user without alias to config')
```
Comment (klas-genestack) [inline genestack_client/settings/genestack_config.py]: 'Cannot add user without alias'

File: genestack_client/scripts/genestack_user_setup.py [hunk 4]
```diff
@@ -99,15 +98,15 @@ def create_user_from_input(host, alias):
...
-        :param host:  server host
-        :type host:  basestring
-        :param alias: user alias
-        :type alias: basestring
-        :return: user
-        :rtype: User
+    Ask email and password, check that it is possible to login with this credentials
+    and return user.
+
+    :param host:  server host
```
Comment (klas-genestack) [inline genestack_client/scripts/genestack_user_setup.py]: Isn't it "server's base URL"?

File: genestack_client/settings/genestack_config.py [hunk 5]
```diff
@@ -83,7 +83,8 @@ def load(self):
         config_path = self.get_settings_file()
 
         if not os.path.exists(config_path):
-            print 'Warning: config is not present. You can setup it via: genestack-user-setup init'
+            sys.stderr.write('Warning: config is not present. '
+                             'You can setup it via: genestack-user-setup init\n')
```
Comment (klas-genestack) [inline genestack_client/settings/genestack_config.py]:
```
Warning: no configuration file found.
Set it up with `genestack-user-setup init`\n
```

File: genestack_client/scripts/genestack_user_setup.py [hunk 6]
```diff
@@ -26,7 +24,8 @@ def validate_alias(alias):
 
 
 def input_alias(existing):
-    print 'Please input alias. (Alias can contain: letters (a-Z), digit (0-9), at (@), underscore (_), hyphen (-))'
+    print ('Please input alias. (Alias can contain: letters (a-Z), '
+           'digit (0-9), at (@), underscore (_), hyphen (-))')
```
Comment (klas-genestack) [inline genestack_client/scripts/genestack_user_setup.py]: Let's make it `letters (a-z, A-Z)` for clarity.

File: genestack_client/scripts/genestack_user_setup.py (hunk 4 above)
Comment (genestack-solomatin) [reply genestack_client/scripts/genestack_user_setup.py]: No, it is host.

File: genestack_client/scripts/genestack_user_setup.py [hunk 7]
```diff
@@ -26,7 +24,8 @@ def validate_alias(alias):
 
 
 def input_alias(existing):
-    print 'Please input alias. (Alias can contain: letters (a-Z), digit (0-9), at (@), underscore (_), hyphen (-))'
+    print ('Please input alias. (Alias can contain: letters (a-z, A-Z), '
+           'digit (0-9), at (@), underscore (_), hyphen (-))')
```
Comment (klas-genestack) [inline genestack_client/scripts/genestack_user_setup.py]: "digits"
"at-sign"

File: genestack_client/settings/genestack_config.py [hunk 8]
```diff
@@ -73,7 +73,7 @@ def add_user(self, user, save=True):
 
     def set_default_user(self, user, save=True):
         if user.alias not in self.__users:
-            raise GenestackException('User %s is not present in config users' % user.alias)
+            raise GenestackException('User "%s" does not exists' % user.alias)
```
Comment (klas-genestack) [inline genestack_client/settings/genestack_config.py]: "does not exist"

**Pull request merged**